experiments/2017-09-21-rust-on-android.md

```diff
@@ -0,0 +1,224 @@
+Following on from the last post on getting a Rust library [building on iOS](https://mozilla.github.io/firefox-browser-architecture/experiments/2017-09-06-rust-on-ios.html), we're now going to deploy the same library on Android.
+
+In order to do Android development, we'll need to set up our Android environment. First we need to install [Android Studio](https://developer.android.com/studio/index.html). Once Android Studio is installed, we'll need to install the NDK (Native Development Kit). 
+
+Open Android Studio. Go to Tools > Tools > Android > SDK Manager > SDK Tools. Check the following options for installation and click `OK`.
+
+    * Android SDK Tools
+    * NDK 
+    * CMake 
+    * LLDB 
+
+Once the NDK and associated tools have been installed, we need to set a few environment variables, first for the SDK path and the second for the NDK path. To find out where  these items are installed, go to File > Project Structure. Here you will find the paths to the SDK and NDK. Set the following envvars:
+```
+export ANDROID_HOME=<SDKPath>
+export NDK_HOME=<NDKPath>
+```
+
+If you do not already have Rust is installed, we need to do this now. For this we will be using [rustup](https://www.rustup.rs/). Rustup installs Rust from the official release channels and enables you to easily switch between different release versions. It will be useful to you for all your future Rust development, not just here.
```

Comment:
s/is //

Reply:
It's worth mentioning that Rustup is worth using even if you have HomeBrew installed.